### ISSUE TITLE
p-dialog does not show

### ISSUE THREAD

**Issue by carolitascl** [primefaces/primeng#2641]
!--
- IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING.
- IF YOU'D LIKE TO SECURE OUR RESPONSE, YOU MAY CONSIDER PRIMENG PRO SUPPORT WHERE SUPPORT IS PROVIDED WITHIN 4 hours.
-->

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please fork the plunkr below and create a case demonstrating your bug report. Issues without a plunkr have much less possibility to be reviewed.

http://plnkr.co/edit/dWh4IJoFdiNDdQVMJwSE

**Current behavior**

Confirm dialogs are not working correctly in version 4 RC5. After the dark screen the dialog does not appear. 

**Expected behavior**

Show dialog

**Minimal reproduction of the problem with instructions**

 You can use the PrimeNg Quickstart Webpack as an example.

https://github.com/primefaces/primeng-quickstart-webpack

Use Angular 2 and Angular 4 to see the problem when you use the "Add" button or click over the grid

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

Webstorm ( latest ), npm ( latest )

* **Angular version:** 2.0.X

Here Works

* **PrimeNG version:** 4 RC 5

Here does not works

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome at least
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

Typescript 2.3.1

* **Node (for AoT issues):** `node --version` =   7.5.0 x64

**Comment by cagataycivici** [comment]
I'm not able to replicate, providing a plunkr would help.

**Comment by Mithi83** [comment]
I ran into a problem that sounds similar. See http://plnkr.co/edit/di7mgd5fAEsw3l9Y5DHr
I just added the dialog code from the example https://www.primefaces.org/primeng/#/dialog to the example plunker you provided.

After looking at the corresponding element I found the following code:
`<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-shadow ui-dialog-draggable" ng-reflect-ng-class="[object Object]" style="display: none; width: 300px; visibility: visible; left: 49.5px; top: 227px; z-index: 1004;">`

Manually editing the "display: none" part to "display: block" solves the problem.
